uber/models.py

```diff
@@ -1182,14 +1182,14 @@ def last_first(cls):
     @property
     def watchlist_guess(self):
         try:
-            with sa.Session() as session:
+            with Session() as session:
                 return session.guess_attendee_watchentry(self)
```

Comment:
Yeah, I took out to_dict because it wasn't iterating properly - it would output multiple loops for a single record.

Are you saying that in this branch you can still reproduce the error, even with the changes?